Learning_test_automation/Projekt#2/Jumanji/GameMechanism/Spawn.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace GameMechanism
+{
+    public class Spawn
+    {
+        private int _high;
+        private int _width;
+        private int column;
+        private int row;
+        private int[] coordinates = new int[2];
+        private List<int[]> organismPosition = new List<int[]>();
```

Comment:
Rozumiem, ze tu trzymasz pozycje jakiegos organizmu. 
Mozesz wziac pod uwage uzycie klasy `Point`, zamiast tablicy int.
Wtedy obiekt bedzie trzymal wspolrzedne x i y ;) 
https://learn.microsoft.com/en-us/dotnet/api/system.drawing.point?view=net-7.0

Reply:
Boszzzeee ile to zmian....